detection-bot/README.md

```diff
@@ -0,0 +1,25 @@
+# Nethermind Bot Deployments Agent
+
+## Description
+
+This agent detects bots that are deployed by Nethermind
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+Describe each of the type of alerts fired by this agent
+
+- NETHERMIND-1
+  - Fired when an occured transaction involves Nethermind's address
+  - Severity is always set to "info"
+  - Type is always set to "info"
+  - Metadata field `address` included with this alert
```

Comment:
```suggestion
## Alerts

- NETHERMIND-1
  - Fired when an occured transaction involves Nethermind's address
  - Severity is always set to "info"
  - Type is always set to "info"
  - Metadata field `address` included with this alert
```
Suggestion to remove unnecessary line. Also recommend updating the bullet point that starts with "Fired when an..." to explain that the bot fires when it detects a bot deployment from the Nethermind deployer address.